src/main/java/core/basesyntax/service/serviceimpl/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.service.FileReader;
+import java.io.File;
```

Comment:
Consider using try-with-resources when working with I/O operations to ensure that all resources are closed properly.